pkg/rancher-desktop/components/RdInput.vue

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  name:         'rd-input',
+  inheritAttrs: false,
+  props:        {
+    value: {
+      type:    [String, Number],
+      default: null,
+    },
+    isLocked: {
+      type:    Boolean,
+      default: false,
+    },
+    tooltip: {
+      type:    String,
+      default: null,
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="rd-input-container">
+    <input
+      :value="value"
+      :disabled="$attrs.disabled || isLocked"
+      v-bind="$attrs"
+      v-on="$listeners"
+    />
+    <slot name="after">
```

Comment:
I'm not sure `after` is the best name for this slot. Maybe `lock` or `icon`?

Reply:
> I assume the incorrect tooltip is expected.

Yeah, I added the translation in #4706. I think we'll make sure to merge that one first so that we will have the complete package with both the custom select and input components.

Reply:
> I'm not sure after is the best name for this slot. Maybe lock or icon?

I find that the flexibility of slots lends to naming that describes layout rather than intended usage. With this in mind, `#after` describes any content that might come after the select element, we just provide a lock icon by default. 

I'm open to renaming to `icon` perhaps, but I don't think `lock` fits for the slot name. 

Alternatively, we can consider removing the slot and declare that we won't need the flexibility (totally possible) and dedicate space solely for the icon where we originally provided the `#after` slot.

Reply:
I'm happy with merging it as-is (hence the PR is still approved), just wanted to double check :)